src/main/resources/META-INF/plugin.xml

```diff
@@ -30,6 +30,13 @@
         <li>Maven</li>
         <li>Gradle</li>
     </ul>
+
+    <h2><strong>Prerequisites:</strong></h2>
+    <p>
+        To use the Snyk Vulnerability Scanner plugin, you must have Synk CLI installed on your machine.
+        <a href="https://support.snyk.io/hc/en-us/articles/360003812538-Install-the-Snyk-CLI">Click here</a>
```

Comment:
Just a nit: it's usually a bad practice to use "[click] here" as a link text. You should describe what the link is - See https://www.w3.org/QA/Tips/noClickHere / https://uxmovement.com/content/why-your-links-should-never-say-click-here/

Reply:
done with 049e8eb